docker/brew.Dockerfile

```diff
@@ -122,25 +122,35 @@ RUN git config --global --add safe.directory ./
 # hadolint ignore=DL3059
 RUN $YARN build --filter=backend
 
+# The special case for imports here is because we need the
+# yarn.lock files of the imported packages to be present
+# so that they can be modified by the cachitoRegistry replacement
+RUN $YARN --cwd ./dynamic-plugins/imports export-dynamic --no-install
+
 # Downstream only - Cachito configuration
 # replace external registry refs with cachito ones
 RUN cachitoRegistry=$(npm config get registry); echo "cachito registry: $cachitoRegistry"; \
```

Comment:
should this be done after all the installs and exports and copies? (line 146)

Reply:
I don't think so.
It would fail when run offline.